editions/tw5.com/tiddlers/nodejs/Installing a plugin - client server.tid

```diff
@@ -0,0 +1,38 @@
+created: 20220611123344385
+modified: 20220612091241651
+tags: [[TiddlyWiki on Node.js]] PluginsCS
+title: Installing a plugin - client server
```

Comment:
This title isn't consistent with others. We hyphenate client-server, so it would be clearer to use the title "Installing official plugins on Node.js"